compiler/rustc_resolve/src/late/diagnostics.rs

```diff
@@ -1508,6 +1508,7 @@ impl<'a: 'ast, 'ast> LateResolutionVisitor<'a, '_, 'ast> {
         Some(match name {
             "byte" => sym::u8, // In Java, bytes are signed, but in practice one almost always wants unsigned bytes.
             "short" => sym::i16,
+            "Boolean" => sym::bool,
```

Comment:
Perhaps would make sense to add "Bool" to this list too? It's common in functional languages like Haskell.

```suggestion
            "Bool" => sym::bool,
            "Boolean" => sym::bool,
```